### PR TITLE
Merge codeAction(s)

### PR DESCRIPTION
**Pull request by mattn**
If you use multiple Language Servers, the codeAction will respond at different times. This change wait all responses to display inputlist() at one time.

### CODE REVIEW

**Comment by mattn** [comment]
Please do not merge until merge https://github.com/prabirshrestha/vim-lsp/pull/735

File: autoload/lsp/ui/vim/code_action.vim [hunk 1]
```diff
@@ -47,48 +51,58 @@ function! lsp#ui#vim#code_action#do(option) abort
...
+        let l:index = inputlist(map(copy(l:total_code_actions), { i, action ->
                     \   printf('%s - %s', i + 1, action['title'])
                     \ }))
     endif
 
     " Execute code action.
-    if 0 < l:index && l:index <= len(l:code_actions)
-        call s:handle_one_code_action(a:server_name, a:sync, a:bufnr, l:code_actions[l:index - 1])
+    if 0 < l:index && l:index <= len(l:total_code_actions)
+        call s:handle_one_code_action(a:server_name, a:sync, a:bufnr, l:total_code_actions[l:index - 1])
```
Comment (hrsh7th) [inline autoload/lsp/ui/vim/code_action.vim]: This server_name is always last respond server's name.
Is it intented?

Comment (mattn) [reply autoload/lsp/ui/vim/code_action.vim]: Thank you. Fixed.
image

Comment (hrsh7th) [reply autoload/lsp/ui/vim/code_action.vim]: Looks awesome!
Thanks for your fix.

**Pull request merged**